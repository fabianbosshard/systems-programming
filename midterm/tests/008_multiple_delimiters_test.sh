exec "$1" 'format=@b @a' delim='|/' <<EOF
Plum|4/7
Lemon|2/5
EOF
