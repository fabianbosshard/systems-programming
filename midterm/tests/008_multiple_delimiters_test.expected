4 Plum
2 Lemon
